src/MobileUI/ViewModels/TopBarViewModel.cs

```diff
@@ -73,19 +76,22 @@ private void SetDefaultAvatar()
         ShowDone = false;
         ShowBack = false;
         ShowAvatar = true;
+        ShowScanner = true;
     }
 
     private void SetBackButton()
     {
         ShowDone = false;
         ShowBack = true;
         ShowAvatar = false;
+        ShowScanner = false;
     }
 
     private void SetDoneButton()
     {
         ShowDone = true;
         ShowBack = false;
         ShowAvatar = false;
+        ShowScanner = true;
```

Comment:
Do you think we need the QR code button on the Quiz result page?
Apart from that, all good 👍

Reply:
Yeah probably not. Set this to false.